cartaya/5/task5.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style5.css">
+    <link rel="stylesheet" href="font-awesome.min.css">
+    <title>Task#5</title>
+</head>
+<body>
+    <header>
+        <div class="language-selector">

```

Comment:
You need always use an convention to the names in all classes, it should create collisions.